core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -375,8 +394,11 @@ class MapUnit {
 
     private fun adjacentHealingBonus(): Int {
         var healingBonus = 0
-        if (hasUnique("This unit and all others in adjacent tiles heal 5 additional HP per turn")) healingBonus += 5
-        if (hasUnique("This unit and all others in adjacent tiles heal 5 additional HP. This unit heals 5 additional HP outside of friendly territory.")) healingBonus += 5
+        healingBonus += getMatchingUniques("All adjacent units heal [] additional HP when healing").sumBy { it.params[0].toInt() }
```

Comment:
Change to "All adjacent units heal [] HP when healing" - that way we won't get the jarring "All adjacent units heal [-5] additional HP when healing"